docs/concepts/privacy.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Privacy
+sidebarDepth: 0
+description: Learn about user privacy in IPFS and why it does not come with a built-in privacy layer or encryption.
+related:
+  'Article: IPFS Privacy (Textile)': https://medium.com/pinata/ipfs-privacy-711f4b72b2ea
+---
+
+# IPFS and privacy
+
+As you explore IPFS and learn about its core concepts, it's important to know that as a protocol for peer-to-peer data storage and delivery, IPFS in itself does not explicitly have a "privacy layer" built-in. This is in line with key principles of protocol design since different uses of IPFS over its life may call for different approaches to privacy. Explicitly implementing an approach to privacy within the IPFS core could "box in" future builders and implementers due to a lack of modularity, flexibility, and future-proofing.
+
+By design, IPFS is a public network: Nodes participating in the network store data affiliated with globally consistent [content addresses](/concepts/content-addressing) (CIDs) and advertise that they have those CIDs available for other nodes to use through publicly viewable [distributed hash tables](/concepts/dht/) (DHTs). Indeed, this is one of IPFS's core strengths — at its most basic, it's a sort of globally distributed "server" of the network's total available data, referenceable both by the content itself (those CIDs) and by the participants who have or want the content (the nodes).
+
+What this does mean, however, is that IPFS itself isn't explicitly protecting knowledge _about_ CIDs and the nodes that provide or retrieve them. This isn't something unique to the distributed web; on both the d-web and the legacy web, traffic, and other metadata can be monitored in ways that can infer a lot about a network and its users. Some key details on this are outlined below, but in summary: IPFS traffic, including which nodes are retrieving and/or reproviding which CIDs, occurs in the clear. If you're worried about the implications of this for your own personal use case, it's worth taking additional measures such as encrypting sensitive content, accessing IPFS content [through Tor gateways](https://dweb-primer.ipfs.io/avenues-for-access/tor-gateways), or even running a private IPFS network if that's appropriate for you.
```

Comment:
> **IPFS traffic,** including which nodes are retrieving and/or reproviding which CIDs, **occurs in the clear**

I wonder if  we need to clarify this in the sentence above and the `::: tip` below.

 I am aware this is a mental shortcut for non-technical reader to highlight that activity may be monitored by other peers, but "in the clear" part will confuse technical ones a lot: they either think "no encryption? wtf!" or know the traffic between peers is encrypted, so not "in the clear", and get bit confused. 

Not sure if we can have cookie and eat cookie, but i'd like to highlight that (1) p2p traffic is encrypted (2) (meta)data is public because anyone can participate in p2p exchanges

How about:

> IPFS traffic between peers is encrypted, however metadata that is published to DHT is public. When IPFS node participates in the public network, it announces CID of data in its cache, own PeerID and addresses on DHT and due to this other peers can learn which nodes are retrieving and/or reproviding which CIDs.

Reply:
How about this:

> IPFS traffic is encrypted. However, when an IPFS node interacts with the public network it announces the CIDs it has stored. This makes it possible for other peers to learn which nodes are providing and requesting which CIDs.

Reply:
I believe https://github.com/ipfs/ipfs-docs/pull/554/commits/0a1e8565f8582d811ce12c33f6e1f0615f727e50 takes care of this, but would appreciate one more look from @lidel if possible.

Also reused the term "PeerID" in the rest of the guide, since I realize I wasn't explicit enough about that term of art.